### ISSUE TITLE
Script install_nimble.nims installs bleeding edge

### ISSUE THREAD

**Issue by davidsblog** [nim-lang/Nim#5153]
I think that the `install_nimble.nims` script installs the bleeding edge version of Nimble, perhaps it would be better if it installed the latest tagged release or some other targeted version.

Today, I used the `install_nimble.nims` from v0.15.2 of Nim and got:
`nimble -v`
`nimble v0.7.11 compiled at 2016-12-27 12:42:35`

Then, later I cloned and compiled the latest Nimble from GitHub and got this:
`src/nimble -v`
`nimble v0.7.11 compiled at 2016-12-27 15:59:10`

In both cases I seem to have gotten v0.7.11.

**Comment by Araq** [comment]
True, but ``install_nimble.nims`` is deprecated anyway and the ``koch.nim`` replacement can do both. (Though it's not documented yet how that works.)

**Comment by davidsblog** [comment]
Yeah, understood, thanks. Although I don't think you can use `koch.nim` to install Nimble from the current *released* version of Nim, can you?  Or to put it another way, is `install_nimble.nims` deprecated in the current release version (0.15.2)?

Although I have used `koch.nim` to install Nimble from the development branch and it worked fine (there was a comment in the output of `install_nimble.nims` explaining what to do).  I'd be very happy to use `koch.nim` if it's available in a stable release.  Thanks again.

**Comment by Araq** [comment]
True, but 0.16 is just around the corner.

**Comment by davidsblog** [comment]
Well, that sounds awesome :) In which case I'd be happy to wait for 0.16 and use `koch` to install Nimble ... (as long as *that* doesn't install the bleeding edge version of Nimble).
